lbrynet/core/client/ConnectionManager.py

```diff
@@ -142,37 +142,21 @@ def _send_secondary_requests(self, peer, protocol):
         ]
         return defer.DeferredList(ds)
 
-    def protocol_disconnected(self, peer, protocol):
-        log.debug("%s protocol %s disconnected for %s",
-                    self._get_log_name(), type(protocol).__name__, peer)
-        if peer in self._peer_connections:
-            del self._peer_connections[peer]
-        if peer in self._connections_closing:
-            d = self._connections_closing[peer]
-            del self._connections_closing[peer]
-            d.callback(True)
-
-    def manage(self):
-        self._manage_deferred = self._manage()
-        if not self.stopped:
-            self._next_manage_call = self.callLater(self.MANAGE_CALL_INTERVAL_SEC, self.manage)
-
-
     @defer.inlineCallbacks
```

Comment:
Instead of using manage() and _manage() so that we can call _manage() in unit tests without scheduling. I added an argument to manage(), schedule_next_call that can be set to False for unit testing and is by default True